lcm-cmake/cpack.cmake

```diff
@@ -6,31 +6,28 @@ set(CPACK_GENERATOR TGZ STGZ)
 
 # Detect OS type, OS variant and target architecture
 if(UNIX)
-    if(APPLE)
-        set(OS_TYPE macos)
-    else()
-        set(OS_TYPE linux)
-
-        # Determine distribution
-        execute_process(COMMAND lsb_release -si
-            OUTPUT_VARIABLE LINUX_DISTRO
-            OUTPUT_STRIP_TRAILING_WHITESPACE
-        )
-
-        # Add CPack generator according to detected distribution
-        if(LINUX_DISTRO STREQUAL "Debian" OR LINUX_DISTRO STREQUAL "Ubuntu" OR LINUX_DISTRO STREQUAL "LinuxMint")
-            list(APPEND CPACK_GENERATOR DEB)
-        endif()
-    endif()
-
-    # Determine architecture
+  if(APPLE)
+    set(OS_TYPE macos)
+  else()
+    set(OS_TYPE linux)
+  endif()
+  # Determine architecture
```

Comment:
Can we not lost the blank lines between different functional sections? That's non-optimal from a readability standpoint...